app/views/admin/admin_users/_user_history.html.erb

```diff
@@ -24,6 +26,28 @@
       <td><%= ts("Account Created") %></td>
       <td><%= ts("System Generated") %></td>
     </tr>
+    <tr>
+      <td>
+        <% if @user.current_sign_in_at.nil? %>
+          <%= ts('No logins recorded') %>
```

Comment:
You should use [double quotes](https://github.com/otwcode/otwarchive/wiki/Coding-standards#general-code-formatting) for all the new additions.

Reply:
For consistency, you should use "No login recorded" (singular). We have "No address recorded" and "No IP recorded" elsewhere.